appinfo/info.xml

```diff
@@ -16,12 +16,12 @@
     <category>tools</category>
     <website>https://github.com/nextcloud/polls</website>
     <bugs>https://github.com/nextcloud/polls/issues</bugs>
+    <use-migrations>true</use-migrations>
```

Comment:
@dartcafe do we need this? I think the migrations work without it. Only oC uses that. Travis fails because this tag is not listed in https://apps.nextcloud.com/schema/apps/info.xsd.

Reply:
Ah. Ok. Thanks for the hint.